go.mod

```diff
@@ -56,7 +56,7 @@ require (
 	github.com/sirupsen/logrus v1.4.1
 	github.com/stretchr/testify v1.3.0
 	github.com/syndtr/gocapability v0.0.0-20180916011248-d98352740cb2 // indirect
-	github.com/tonistiigi/fsutil v0.0.0-20190819224149-3d2716dd0a4d
```

Comment:
This commit had module related errors when resolving `golang.org/x/crypto`; updating fixes it, and the changes from 3d2716dd0a4d to 0f039a052ca1 seem to be fairly minor.